src/postinstall.js

```diff
@@ -1,20 +1,31 @@
+import fs from 'node:fs';
 import process from 'node:process';
+import path from 'node:path';
 
 import semver from 'semver';
 
 import get from './get.js';
 import util from './util.js';
 
-import nodeManifest from '../package.json' assert { type: 'json' };
-
 await postinstall()
     .catch((error) => {
         if (error.code === 'EPERM') {
             console.error('Unable to create symlink since user did not run as Administrator.');
+        } else {
+            console.error(error)
         }
     });
 
 async function postinstall() {
+    
+    /**
+     * @type {fs.PathLike}
+     */
+    const nodeManifestPath = path.resolve('package.json');
```

Comment:
The previous import was navigating one directory level up (`import '../package.json'`), but this path does not include `..` to perform the same directory level navigation.

Is this by intention?

Reply:
Yes. When using module import, the file path is relative to that module. When using readFile, the file path is relative from pwd.

Reply:
Actually let me test and get back to you on this.

Reply:
I had to use __dirname to get the current dir the module is in and traverse one dir up. This is probably why tests are failing